<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>695</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>695</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>695</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>962</width>
      <height>640</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>962</width>
      <height>640</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>962</width>
      <height>640</height>
     </size>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="imageLabel_ori">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>450</width>
       <height>450</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>450</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>450</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="imageLabel_segmented">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>170</y>
       <width>450</width>
       <height>450</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>450</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>450</height>
      </size>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>601</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Non-local Means Denoising</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>h</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>template window size</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>search windows size</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_nlH">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_nlTemplate">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>40</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_nlSearch">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>40</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>80</y>
       <width>311</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="title">
      <string>CLAHE</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>limit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>size</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_claheLimit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_claheSize">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>40</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>931</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="title">
      <string>General</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Gamma</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Median filter size</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>30</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Trim size</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_trim">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_median">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_gamma">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_threshold">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget_images">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>211</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_process">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>430</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_accuracy">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>570</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Database Accuracy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>620</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Accuracy  = </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_accuracy">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>620</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00.00 %</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_opticdisc">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>500</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Find Optic Disc</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuParams">
    <property name="title">
     <string>Params</string>
    </property>
    <addaction name="actionDRIVE_Params"/>
    <addaction name="actionCHASE_Params"/>
    <addaction name="actionSTARE_Params"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Default_Params"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuParams"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Import Dataset</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDRIVE_Params">
   <property name="text">
    <string>Load DRIVE Params</string>
   </property>
  </action>
  <action name="actionSTARE_Params">
   <property name="text">
    <string>Load STARE Params</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCHASE_Params">
   <property name="text">
    <string>Load CHASE Params</string>
   </property>
  </action>
  <action name="actionLoad_Default_Params">
   <property name="text">
    <string>Load Default Params</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
